pkg/apis/servicecatalog/types.go

```diff
@@ -510,6 +519,26 @@ const (
 	ServiceInstanceOperationDeprovision ServiceInstanceOperation = "Deprovision"
 )
 
+// ServiceInstancePropertiesState is the state of a ServiceInstance that
+// the ServiceBroker knows about.
+type ServiceInstancePropertiesState struct {
+	// PlanName is the name of the plan that the broker knows this
+	// ServiceInstance to be on. This is the human readable plan name from the
+	// OSB API.
+	PlanName string
```

Comment:
I'd prefer to call it ExternalServicePlanName just like what we decided to call it on ServiceInstance.Spec for consistency.

Reply:
+1 to that